### PR TITLE
feat(expandable): add expandable - INNO-1288

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

Add expandable, based on [specifications](https://webgate.ec.europa.eu/CITnet/confluence/display/NEXTEUROPA/Expandable+-+Design+specifications)

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit b1bf26ca6c4dc4ce6988c9285ca0ef0b21a77d77

https://deploy-preview-973--europa-component-library.netlify.com

File: src/systems/ec/implementations/vanilla/packages/ec-component-expandable/ec-component-expandable.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+@import '@ecl/ec-base/ec-base';
+
+@mixin ecl-expandable() {
+  .ecl-expandable__toggle[aria-expanded='true'] {
+    [data-ecl-icon='true'] {
+      transform: rotate(0deg);
+    }
+  }
+
+  :not(.no-js) .ecl-expandable__content[aria-hidden='true'] {
```
Comment (yhuard) [inline src/systems/ec/implementations/vanilla/packages/ec-component-expandable/ec-component-expandable.scss]: instead of `:not(.no-js)`, you can use `.js`

**Review by yhuard — COMMENTED** [review]
PR on hold

**Review by yhuard — CHANGES_REQUESTED** [review]
there might have been a misunderstanding: an expandable never involves a link (it's not the purpose of a link to trigger expandables)

**Review by yhuard — CHANGES_REQUESTED** [review]
one last thing and then I think we're good: React packages must be private (see https://github.com/ec-europa/europa-component-library/blob/v2/src/systems/ec/implementations/react/packages/button/package.json)

**Comment by emeryro** [comment]
React packages are now private

**Review by yhuard — APPROVED** [review]

**Pull request merged**